ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAP-600 Add first court hearing details to CYA

## Description of change
Follow-up to previous PRs #493 and #494 to add the first court hearing details (if any) to the check your answers page.

Also bump to the schemas version where this new attributes are part of the Struct to be able to read back submitted applications.

If the first court hearing question was not asked, we hide the row and section.

It is important to merge this datastore PR too otherwise when reading submitted applications will not have the new attributes: https://github.com/ministryofjustice/laa-criminal-applications-datastore/pull/175

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-600

## Notes for reviewer

## Screenshots of changes (if applicable)
**When there is a different first court hearing**
<img width="820" alt="Screenshot 2023-10-03 at 15 31 27" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/687910/4d9070cd-f0a4-44df-a258-be711434c3f4">

**When first court hearing is the same as next court hearing**
<img width="792" alt="Screenshot 2023-10-03 at 15 31 42" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/687910/5ad23aa4-d71a-43d2-8126-bbb703198483">


## How to manually test the feature
In localhost, you will need to also use the datastore in the branch above mentioned in order to render submitted applications.
For in progress applications, go to the next court hearing page, and select different options to the question "Did this court also hear the first hearing?" and fill the first court hearing if the answer is "No". Observe the CYA page will show/hide this accordingly.
